ckanext/subscribe/action.py

```diff
@@ -38,13 +60,32 @@ def subscribe_signup(context, data_dict):
     :param skip_verification: Doesn't send email - instead it marks the
         subscription as verified. Can be used by sysadmins only.
         (optional, default=False)
+    :param g-recaptcha-response: Recaptcha response from frontend signup form
```

Comment:
I would rename this with underscores instead of dashes if we are using it as a param in the data_dict, and not mention here that it comes from the frontend signup form, because it doesn't necessarily come from there, right?